app/views/internalServerError.scala.html

```diff
@@ -1,4 +1,4 @@
-@()(implicit messages: Messages, request: RequestHeader)
+@(name: String)(implicit messages: Messages, request: RequestHeader)
```

Comment:
It doesn't look like this variable is used in the file. It just needs passing into the `@main`  method I think.